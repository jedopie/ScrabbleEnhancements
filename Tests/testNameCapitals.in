1
a
A
b
