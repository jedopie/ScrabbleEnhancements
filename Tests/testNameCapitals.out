Welcome to Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> Starting a New Game 

Enter a name for Player 1 (Uppercase characters only)
> Sorry, please enter a name with uppercase characters


Enter a name for Player 2 (Uppercase characters only)
> Sorry, please enter a name with uppercase characters
